ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Selection-Tool/bond-properties.spec.ts

```diff
@@ -0,0 +1,607 @@
+/* eslint-disable no-magic-numbers */
+import { test, expect, Page } from '@playwright/test';
+import {
+  BondType,
+  doubleClickOnBond,
+  dragMouseTo,
+  getCoordinatesOfTheMiddleOfTheScreen,
+  LeftPanelButton,
+  openFileAndAddToCanvas,
+  pressButton,
+  receiveFileComparisonData,
+  RingButton,
+  saveToFile,
+  selectAllStructuresOnCanvas,
+  selectLeftPanelButton,
+  selectRingButton,
+  takeEditorScreenshot,
+} from '@utils';
+import { getMolfile, getRxn } from '@utils/formats';
+
+async function selectDoubleOption(page: Page) {
+  await page.getByText('Single').click();
+  await page.getByRole('option', { name: 'Double', exact: true }).click();
+}
+
+test.describe('Bond Properties', () => {
```

Comment:
I would suggest to make it more common by adding more arguments to the function.
```
async function selectDoubleOption(page: Page, param1, param2) {
  await page.getByText(param1).click();
  await page.getByRole('option', { name: param2 exact: true }).click();
}
```
It will allow us to reduce more duplicated code.
If you agree please choose appropriate name for  param1, param2

Reply:
Now it looks better, tnx